src/mixed-line-bar-chart/chart-container.tsx

```diff
@@ -372,6 +376,13 @@ export default function ChartContainer<T extends ChartDataTypes>({
       if (isPopoverPinned) {
         dismissPopover();
       } else {
+        // If the focus is on the document body when pinning the popover,
+        // Safari will focus any interactive element that may exist inside the popover body
+        // right *after* we focus the dismiss button.
+        // We prevent this here by focusing the chart plot before pinning the popover.
+        // For the purpose of this workaround, we could focus anything except the document body
+        // --the focus will move to the dismiss button once the popover appears in the next frame, anyway.
+        plotRef.current?.focusApplication();
```

Comment:
Is there any reference why Safari does that? Can we do this change as a separate PR (we do already allow interactive content in the popover so the fix makes sense on its own it seems)?

Reply:
Opened a separate PR: https://github.com/cloudscape-design/components/pull/1784 and removed these changes from this one.